docs/AsyncWrap/README.md

```diff
@@ -1,15 +1,276 @@
 Node.js tracing - AsyncWrap
 ================================================================================
 
-AsyncWrap is the central location in Node.js that handles async things
-specific to Node.js.  As such, it's a likely place where tracing will be
-desired.  Should serve as a good customer of the more general tracing story for
-Node.js.
+AsyncWrap is two things. One is a
+[class abstraction](https://github.com/nodejs/node/blob/master/src/async-wrap.h)
+there provides node internal facilities for handling async things, such as
```

Comment:
"that provides an internal mechanism for handling asynchronous tasks, such as..."